app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,25 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(
+        status: :failed,
+        metadata: "Reset by admin",
+      )
+      Export.create_and_enqueue!(snap_application: application, destination:
+                                 :fax)
```

Comment:
did you see my previous comment about style? It's small but I do think this format is harder to read than
```
Export.create_and_enqueue!(
  snap_application: application,
  destination: :fax,
)
```